src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -0,0 +1,361 @@
+        pgm_sym_gens["node"] = self._get_ids("bus", pp_sgens["bus"])
+        pgm_sym_gens["status"] = pp_sgens["in_service"]
+        pgm_sym_gens["p_specified"] = pp_sgens["p_mw"] * 1e6
+        pgm_sym_gens["q_specified"] = pp_sgens["q_mvar"] * 1e6
+
+        self.pgm_data["sym_gen"] = pgm_sym_gens
+
+    def _create_pgm_input_sym_loads(self):
+        assert "sym_load" not in self.pgm_data
+
+        pp_loads = self.pp_data["load"]
+
+        n_loads = len(pp_loads)
+
+        pgm_sym_loads = initialize_array(data_type="input", component_type="sym_load", shape=3 * n_loads)
+
+        const_i_multiplier = pp_loads["const_i_percent"] * (1e-2 * 1e6)
+        const_z_multiplier = pp_loads["const_z_percent"] * (1e-2 * 1e6)
+        const_p_multiplier = 1e6 - const_i_multiplier - const_z_multiplier
+
+        pgm_sym_loads["id"][:n_loads] = self._generate_ids("sym_load_const_power", pp_loads.index)
+        pgm_sym_loads["node"][:n_loads] = self._get_ids("bus", pp_loads["bus"])
+        pgm_sym_loads["status"][:n_loads] = pp_loads["in_service"]
+        pgm_sym_loads["type"][:n_loads] = LoadGenType.const_power
+        pgm_sym_loads["p_specified"][:n_loads] = const_p_multiplier * pp_loads["p_mw"]
+        pgm_sym_loads["q_specified"][:n_loads] = const_p_multiplier * pp_loads["q_mvar"]
+
+        pgm_sym_loads["id"][n_loads : 2 * n_loads] = self._generate_ids("sym_load_const_impedance", pp_loads.index)
+        pgm_sym_loads["node"][n_loads : 2 * n_loads] = self._get_ids("bus", pp_loads["bus"])
+        pgm_sym_loads["status"][n_loads : 2 * n_loads] = pp_loads["in_service"]
+        pgm_sym_loads["type"][n_loads : 2 * n_loads] = LoadGenType.const_impedance
+        pgm_sym_loads["p_specified"][n_loads : 2 * n_loads] = const_z_multiplier * pp_loads["p_mw"]
+        pgm_sym_loads["q_specified"][n_loads : 2 * n_loads] = const_z_multiplier * pp_loads["q_mvar"]
+
+        pgm_sym_loads["id"][-n_loads:] = self._generate_ids("sym_load_const_current", pp_loads.index)
+        pgm_sym_loads["node"][-n_loads:] = self._get_ids("bus", pp_loads["bus"])
+        pgm_sym_loads["status"][-n_loads:] = pp_loads["in_service"]
+        pgm_sym_loads["type"][-n_loads:] = LoadGenType.const_current
+        pgm_sym_loads["p_specified"][-n_loads:] = const_i_multiplier * pp_loads["p_mw"]
+        pgm_sym_loads["q_specified"][-n_loads:] = const_i_multiplier * pp_loads["q_mvar"]
+
+        self.pgm_data["sym_load"] = pgm_sym_loads
+
+    def _create_pgm_input_transformers(self):
+        assert "transformer" not in self.pgm_data
+
+        pp_trafo = self.pp_data["trafo"]
+
+        pgm_transformers = initialize_array(data_type="input", component_type="transformer", shape=len(pp_trafo))
+        pgm_transformers["id"] = self._generate_ids("transformer", pp_trafo.index)
+        pgm_transformers["from_node"] = self._get_ids("bus", pp_trafo["hv_bus"])
+        pgm_transformers["from_status"] = pp_trafo["in_service"]
```

Comment:
Should add switches here similar to line.

Reply:
I should add them to three winding transformer as well then, right?

Reply:
Added them to transformer and three winding transformer.